inc/saiacl.h

```diff
@@ -396,12 +396,26 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_START,
 
     /**
+     * @brief Available ACL entries for this table
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_ACL_TABLE_ATTR_AVAILABLE_ACL_ENTRY = SAI_ACL_TABLE_ATTR_START,
+
+    /**
+     * @brief Available ACL counters for this table
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_ACL_TABLE_ATTR_AVAILABLE_ACL_COUNTER,
```

Comment:
one acl table can be bind to different binding ports, there may be multiple instances of such table and each instance may be its own limit. Therefore, we should add this attribute in the acl group member.